mobile/src/profile/styles.ts

```diff
@@ -87,30 +90,31 @@ export const contentStyles = StyleSheet.create({
     position: {
         fontSize: 13,
         textAlign: 'center',
-        color: '#000'
+        color: '#000',
+        top: 10
     },
     department: {
         fontSize: 13,
         textAlign: 'center',
         color: '#2FAFCC',
-        fontWeight: 'bold'
+        fontWeight: 'bold',
+        top: 15
     },
     infoContainer: {
         flexDirection: 'row',
-        padding: 5
+        padding: paddingInfoContainer,
+        top: 30
     },
     contactsContainer: {
-        paddingTop: 20,
+        top: 40,
         padding: 5,
-        alignItems: 'center',
-        justifyContent: 'center'
     }
 });
 
 export const tileStyles = StyleSheet.create({
     container: {
-        padding: 1,
-        width: '25%'
+        padding: paddingTile,
+        width: (Dimensions.get('window').width - 2 * paddingInfoContainer - 3 * paddingTile) / 4,
```

Comment:
What is wrong with 25%?

Reply:
The problem is with pixels between tiles, i will try what Sergey has suggested.